src/main/java/no/nav/veilarbarena/controller/v2/ArenaV2Controller.java

```diff
@@ -108,6 +108,16 @@ public AktiviteterDTO hentAktiviteterV2(@RequestBody PersonRequest personRequest
                 .map(this::mapArenaAktiviteter)
                 .orElseThrow(() -> new ResponseStatusException(HttpStatus.NO_CONTENT));
     }
+
+    /**
+     * Registrer ikke-arbeidssøker i arena
+     */
+    @PostMapping("/registrer-ikke-arbeidssoker")
+    public String registrerIkkeArbeidssoker(@RequestBody PersonRequest personRequest) {
+        authService.sjekkTilgang(personRequest.getFnr());
```

Comment:
Her må veileder ha tilgang til person for å kunne starte oppfølging. Er dette korrekt?